vendor/bat-native-ads/src/bat/ads/internal/ads_impl.h

```diff
@@ -150,6 +150,7 @@ class AdsImpl : public Ads {
   bool IsAllowedToShowAds();
   bool DoesHistoryRespectMinimumWaitTimeToShowAds();
   bool DoesHistoryRespectAdsPerDayLimit();
+  void CleanPostedAsRegistryAfterReboot();
```

Comment:
Can we please relocate closer to the notification function calls, i.e. `StartDeliveringNotifications` etc.

Reply:
Resolving as function no longer exists so will review latest changes